samples/deployment-manager/deploy-forseti.yaml.sample

```diff
@@ -17,13 +17,13 @@
 
 # Change these import paths as necessary to suit your environement.
 imports:
-- path: py/inventory/cloudsql-instance.py
+- path: ../../deployment-templates/inventory/cloudsql-instance.py
```

Comment:
Deployment manager doesn't like absolute path. I suspect because it doesn't know the root to start looking from.